src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -111,6 +114,8 @@ def __init__(self, cmd: AzCommandsLoader, resource_type: ResourceType):
         )
         # init nat gateway models
         self.init_nat_gateway_models()
+        # init pod identity models
+        self.init_pod_identity_models()
```

Comment:
question: what's the differences between L118 and L100?

```python
        self.ManagedClusterPodIdentityProfile = self.__cmd.get_models(
            "ManagedClusterPodIdentityProfile",
            resource_type=self.resource_type,
            operation_group="managed_clusters",
        )
```

Reply:
No substantial changes. Just put the attribute of the original class into a dictionary. In those functions that need to use model definitions, we will pass the dictionary as a parameter to it to avoid hard-coding the import statement (e.g. from xxxx_xx_xx import xxxx).